visualizations/frontend/other/TTable.vue

```diff
@@ -274,6 +273,10 @@
 export default {
   name: "TTable",
   props: {
+    withBorder: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
I don't love adding a new prop for styling. However, I feel that this strikes a balance between design consistency, and also allowing for a borderless variant.
I am open to other suggestions still.
This could be renamed to `hasBorder` if that works better.